express/parsers/apps/espresso/formats/xml7.py

```diff
@@ -0,0 +1,231 @@
+from typing import Union
+from xml.etree.ElementTree import Element
+
+import numpy as np
+
+from express.parsers.settings import Constant
+
+from .xml import EspressoXMLParser
+
+
+class EspressoXMLParserV7(EspressoXMLParser):
```

Comment:
I think a better approach would be to rename the old parser to `EspressoXMLParserPreV6.4` or something like that for clarity.

The new one could be `ExpressoXMLParserPostVersion6.4`. If they add a new format, we'll rename it to `ExpressoXMLParserPostV6.4PreV7.8` for example.